### ISSUE TITLE
Calendar rows should have role attribute

### ISSUE THREAD

**Issue by davewrightOSF** [acidb/mobiscroll#473]
The calendar widget fails accessibility tests using the AXE chrome extension. This seems to be because the  element with role=grid doesn't have children with role=row (or rowgroup). This is patched by adding the following jQuery...

`$('.mbsc-cal-row').attr('role', 'row');`

But should probably be added to core

**Comment by dioslaska** [comment]
Fixed in 4.10.0
